### PR TITLE
db/data series converter

### PR DESCRIPTION
**Pull request by karolkolodziej**
Moved DataSeriesConverter to the core.

### CODE REVIEW

File: ts/Core/DataSeriesConverter.ts [hunk 1]
```diff
@@ -21,13 +21,13 @@
...
-import type DataEvent from './DataEvent';
+import type DataEvent from '../Data/DataEvent';
 import type LineSeries from '../Series/Line/LineSeries';
 import type PointOptions from '../Core/Series/PointOptions';
 import type SeriesOptions from '../Core/Series/SeriesOptions';
 
-import DataTable from './DataTable.js';
-import U from '../Core/Utilities.js';
+import DataTable from '../Data/DataTable.js';
+import U from './Utilities.js';
```
Comment (bre1470) [inline ts/Core/DataSeriesConverter.ts:30]: The converter should be placed in `ts/Core/Series/`.

**Review by bre1470 — CHANGES_REQUESTED** [review]

**Review by pawellysy — COMMENTED** [review]
Code looks good. Just one question:
Are there any other files, that reference this file?

**Comment by bre1470** [comment]
Probably not. It will eventually be merged with the SeriesTableComposition.

**Review by bre1470 — APPROVED** [review]
Looks good. 👍

**Pull request merged**